src/synthesizer/dust/emission.py

```diff
@@ -450,7 +447,8 @@ def get_spectra(self, _lam, dust_components=False):
         """
 
         if self.template == "DL07":
-            print("Using the Draine & Li 2007 dust models")
+            if verbose:
```

Comment:
should this be a warn?

Reply:
I spent a while thinking about that but it's more of a "statement of fact" than a warning hence why I placed it in a verbose check.